tests/data/annotation_types/geometry/test_rectangle.py

```diff
@@ -16,5 +15,9 @@ def test_rectangle():
     expected['coordinates'] = tuple([tuple([tuple(x) for x in points[0]])])
     assert rectangle.shapely.__geo_interface__ == expected
 
-    raster = rectangle.draw(height=32, width=32)
-    assert (cv2.imread("tests/data/assets/rectangle.png") == raster).all()
+    # raster = rectangle.draw(height=32, width=32)
+    # assert (cv2.imread("tests/data/assets/rectangle.png") == raster).all()
```

Comment:
What happened here?

Reply:
unsure but I'll uncommit to see if it passes